eng/common/scripts/stress-testing/stress-test-deployment-lib.ps1

```diff
@@ -149,8 +149,10 @@ function DeployStressTests(
             -login:$login
     }
 
-    Write-Host "Releases deployed by $deployer"
-    Run helm list --all-namespaces -l deployId=$deployer
+    if ($FailedCommands.Count -lt $pkgs.Count) {
```

Comment:
Reason for this if statement?

Reply:
Normally we print out all releases the user has released (including past releases) at the end of the script. If _none_ of the packages in the current script run have successfully deployed then it's a useless log statement, and it also makes the error logs further up in the history and harder to notice.